src/qt/qtwebkit/Source/WebCore/platform/graphics/qt/ImageBufferQt.cpp

```diff
@@ -258,9 +258,12 @@ void ImageBuffer::putByteArray(Multiply multiplied, Uint8ClampedArray* source, c
 
 static bool encodeImage(const QPixmap& pixmap, const String& format, const double* quality, QByteArray& data)
 {
-    int compressionQuality = 100;
-    if (quality && *quality >= 0.0 && *quality <= 1.0)
-        compressionQuality = static_cast<int>(*quality * 100 + 0.5);
+    int compressionQuality = -1;
+    if (format == "jpeg") {
+        compressionQuality = 100;
```

Comment:
@zackw That was what I would prefer, and it was what I did initially. But I didn't know if 100 was deliberately chosen to be the default value for JPEGs in PhantomJS. I wonder what the default quality is for jpegs in QPixmap.